frontends/PyCDE/src/pycde/module.py

```diff
@@ -238,7 +240,15 @@ def __init__(self, gen_func):
     self.generated_modules = {}
 
   def _generate(self, mod):
-    return self.gen_func(mod, **self.params)
+    gf_args = inspect.getfullargspec(self.gen_func).args
+    call_args = {}
+    for argname in gf_args[1:]:
```

Comment:
So the first argument will always be the module, and the `Inputs` will be accessed through that? And every other argument should be a parameter? This is what I'm seeing, just want to double check.

Reply:
Yes